.eslintrc.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  parser: "babel-eslint",
+  extends: ["plugin:prettier/recommended"],
```

Comment:
Let's use the webex eslint base config:
https://www.npmjs.com/package/@webex/eslint-config-base

Reply:
then we need to add that to `webex-components` too.

Reply:
It already is. [eslint-config-react](https://github.com/webex/components/blob/master/.eslintrc#L4) includes the base config and adds react linting.